packages/ext-angular-modern-boilerplate/src/app/About/about.component.ts

```diff
@@ -0,0 +1,21 @@
+import { Component, OnInit, ViewChild, ElementRef } from '@angular/core';
+
+declare var Ext: any;
+
+@Component({
+  selector: 'app-about',
+  templateUrl: './about.component.html',
+  styles: [``]
+})
+export class AboutComponent implements OnInit {
+
+  constructor() { }
+
```

Comment:
Is all this space intentional? Could it be removed?

Reply:
Fixed.